inscriptions/ISic2933.xml

```diff
@@ -187,11 +192,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
+                <ab>
+                    <lb n="1"/>Hic requie<hi rend="reversed">s</hi>cit in p
+                    <lb n="2" break="no"/>acae <expan><abbr><am><g ref="http://sicily.classics.ox.ac.uk/alists/listglyph.xml#chi-rho">☧</g></am></abbr><ex>risti</ex></expan> infa<supplied reason="omitted">n</supplied><hi rend="reversed">s</hi> Bonipa
```

Comment:
check with Simona, but I think if you put the chi-rho into an 'am' then you need to add 'Ch' to the 'ex' element, or all we will get is 'risti' in the transformation without the chi-rho